src/main/java/frc/robot/subsystems/Revolver.java

```diff
@@ -22,28 +24,12 @@
    */
   private final CANSparkMax m_drumMotor;
   private final CANSparkMax m_feedWheelMotor;
-  private int m_repsAboveDrum = 0;
-  private int m_repsAboveFeed = 0;
-  private final PowerDistributionPanel m_pdp;
+  private int m_drumReps = 0;
+  private int m_feedReps = 0;
```

Comment:
We've decided to no longer check jams on the feeder, only on the revolver. Do you mind removing all the feeder wheel checks?